test/SearchClientTest.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
-namespace CultuurNet\SearchV3\Test;
+namespace CultuurNet\SearchV3;
 
 use CultuurNet\SearchV3\Serializer\SerializerInterface;
-use CultuurNet\SearchV3\SearchQueryInterface;
-use CultuurNet\SearchV3\SearchClient;
-use Guzzle\Http\Message\Response;
+use CultuurNet\SearchV3\ValueObjects\PagedCollection;
 use GuzzleHttp\Client;
 use GuzzleHttp\ClientInterface;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\ResponseInterface;
 
-class SearchClientTest extends \PHPUnit_Framework_TestCase
+class SearchClientTest extends TestCase
 {
     /**
      * @var ClientInterface | \PHPUnit_Framework_MockObject_MockObject
```

Comment:
I think `MockObject` can be used now.

Reply:
Fixed in https://github.com/cultuurnet/search-v3/pull/43/commits/45fc79b88bbc23d7ea6dc4653f88c6cebcdefe0c